### ISSUE TITLE
HeatTempMax Error

### ISSUE THREAD

**Issue by uspino2** [maladr01d/homebridge-venstar-thermostats#2]
Working fine with a ColorTouch, but I keep getting this error. Anything I can do about it? Thanks!

`[12/16/2022, 5:53:51 AM] [homebridge-venstar-thermostats] This plugin generated a warning from the characteristic 'Heating Threshold Temperature': characteristic was supplied illegal value: number 25.6 exceeded maximum of 25. See https://homebridge.io/w/JtMGR for more info.`

**Comment by maladr01d** [comment]
Hi there! So so sorry I missed this. I was new to developing things and
using git and did this more as a personal project so didn't even see your
message from a few years ago. Thank you for notifying me about this bug.
You may have moved on, but I finally updated the plugin and hopefully
addressed this issue.

**Comment by uspino2** [comment]
Thanks for replying. I did move on to the Nest environment. Venstar was ahead of its time years and yers ago, but they just seemed to stop caring about their products. But thanks again, developing plugins is fun and addressing issues helps a lot.

**Comment by maladr01d** [comment]
Yeah, venstars are weird (and sometimes it seems wildly inaccurate) but
they were already installed in my house, so I make them work. Have a good
one! Thanks again.
